# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1284837944 14400
#      Sat Sep 18 15:25:44 2010 -0400
# Node ID 061bd86a0bf3f0dc04ef6d6aa2b9349c61bf43e2
# Parent  88687577519f3a15e7c7875bd90f3ecf2b25746d
Makefile.am (BUILT_SOURCES): include run-octave in the list

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2010-09-18  John W. Eaton  <jwe@octave.org>
+
+	* Makefile.am (BUILT_SOURCES): Include run-octave in the list.
+
 2010-09-16  John W. Eaton  <jwe@octave.org>
 
 	* configure.ac (SCRIPTS_EXE_SUFFIX) Delete variable and all uses.
 	(BUILD_COMPILED_AUX_PROGRAMS): New variable.
 	(AMCOND_BUILD_COMPILED_AUX_PROGRAMS): New Automake conditional.
 	* Makefile.am (SCRIPTS_EXE_SUFFIX, BUILT_SHELL_FILES): Delete
 	variables and all uses.
 	(bin_SCRIPTS, bin_PROGRAMS, mkoctfile_SOURCES,
diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -88,24 +88,28 @@ octave_config_SOURCES =
 nodist_octave_config_SOURCES = octave-config.cc
 
 octave_bug_SOURCES =
 nodist_octave_bug_SOURCES = octave-bug.cc
 
 BUILT_SOURCES = \
   mkoctfile.cc \
   octave-config.cc \
-  octave-bug.cc
+  octave-bug.cc \
+  run-octave
 
 else
 bin_SCRIPTS = \
   mkoctfile \
   octave-config \
   octave-bug \
   run-octave
+
+BUILT_SOURCES = \
+  run-octave
 endif
 
 noinst_SCRIPTS = run-octave
 
 INFO_FILES = \
   INSTALL.OCTAVE \
   BUGS
 
